jobserver/urls.py

```diff
@@ -189,6 +189,16 @@
     path("<str:pk>/<path:path>", ReleaseDetail.as_view(), name="release-detail"),
 ]
 
+status_urls = [
+    path("", Status.as_view(), name="status"),
+    path("<slug:backend>/", PerBackendStatus.as_view(), name="status-backend"),
+    path(
+        "<slug:backend>/db-availability/",
```

Comment:
I'd be inclined to use the term `maintenance` rather than `availability`. The former has a more specific documented meaning, and is "business as usual", where as the db being "unavailable" is arguably a separate check and a more serious problem

Reply:
From the slack discussion we've chosen to keep this as is to avoid `GET 200 /maintenance` meaning that db maintenance is off and causing confusion, but we all agree we don't like it.